backend/src/test/java/com/github/redstylzz/backend/service/CategoryServiceTest.java

```diff
@@ -78,20 +81,44 @@ void shouldReturnListOnSuccessfulRename() {
         when(repository.findByUserIDAndCategoryID(anyString(), anyString())).thenReturn(TestDataProvider.testCategory());
         when(repository.existsByUserIDAndCategoryName(anyString(), anyString())).thenReturn(false);
 
-        underTest.renameCategory(user, "", "");
+        List<Category> wantedList = underTest.renameCategory(user, "", "");
 
         verify(repository).save(any(Category.class));
         verify(repository).findAllByUserID(anyString());
-
+        assertEquals(List.of(), wantedList);
```

Comment:
when using Hamcrest it would be easier to read:
assertThat(myList, is(empty()));